packages/openapi-fetch/src/index.js

```diff
@@ -81,26 +116,17 @@ export default function createClient(clientOptions) {
     if (response.ok) {
       // if "stream", skip parsing entirely
       if (parseAs === "stream") {
-        // fix for bun: bun consumes response.body, therefore clone before accessing
-        // TODO: test this?
-        return { data: response.clone().body, response };
```

Comment:
I removed a lot of the `.clone()`-ing that was happening internally. I profiled this, and this can be a pretty big bottleneck, in addition to a waste of memory for large responses.

Instead, I’d rather users learn about safe vs unsafe `body` access rather than shield them from this. `openapi-fetch` will consume the body by calling `.json()` automatically (because 99% of the time that’s what users want). And if users need to read the raw body, then they’re doing something advanced and should know to `.clone()`.

Reply:
This related Bun issue here (#1486) I think is a mixture of a potential bug in Bun (which may be patched by now) along with user error. In either case, will investigate separately rather than cause potential performance bottlenecks for everyone.